subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -2484,9 +2484,7 @@ static inline u8_t isr_rx_conn_pkt_ctrl_dle(struct pdu_data *pdu_data_rx,
 	     * with response.
 	     */
 	    ((_radio.conn_curr->llcp_length.req ==
-	      _radio.conn_curr->llcp_length.ack) &&
-	     (pdu_data_rx->llctrl.opcode ==
-	      PDU_DATA_LLCTRL_TYPE_LENGTH_REQ)) ||
+	      _radio.conn_curr->llcp_length.ack) && node_tx) ||
```

Comment:
Same comment as in #23707; IMHO the original was easier to read

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/23707#discussion_r399163568